packages/tooling/fast-tooling-wasm/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@microsoft/fast-tooling-wasm",
+  "private": true,
+  "version": "1.0.0",
+  "description": "A set of tooling utilities which has a WASM compile target",
+  "scripts": {
+    "build": "yarn build:permutator",
```

Comment:
do we need "build:permutator"?  Seems like both commands run the same script so why not just use "build"?

Reply:
If you see the contributing doc, this is for multiple projects not just the permutator, each will have it's own build script and there's a process for adding a project to that effect.

Reply:
Haven't we in the past not added things until they were required?

Reply:
This is to make contributing easier which I think is important. Otherwise we will have contributors trying to figure out how to add their project to the rest of the scaffolding whereas here I can be relatively prescriptive.